### PR TITLE
Moe/connect visuals

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: app/components/shared/EtoFormProgressWidget.module.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .eto-form-progress-widget {
-  @include rect(300px);
+  @include rect(px-to-rem(280px));
```
Comment (krzkaczor) [inline app/components/shared/EtoFormProgressWidget.module.scss]: why we are chaning that?

Comment (Moejoe90) [reply app/components/shared/EtoFormProgressWidget.module.scss]: it looks not responsive at all so we need to have a majour fix i can rever it back anyway

File: app/components/eto/dashboard/ETOFormsProgressSection.tsx [hunk 2]
```diff
@@ -42,50 +46,59 @@ class ETOFormsProgressSectionWidget extends React.Component<IProps> {
...
         </Col>
-        <Col md={4} className="pt-2">
-          <Link to={etoRegisterRoutes.keyIndividuals}>
-            <Button>Key individuals</Button>
-          </Link>
+        <Col lg={4} xs={12} className="pt-2">
+          <EtoFormProgressWidget
+            isLoading={loadingData}
+            to={etoRegisterRoutes.keyIndividuals}
+            progress={shouldEtoDataLoad ? EtoKeyIndividualsProgress : 0}
```
Comment (krzkaczor) [inline app/components/eto/dashboard/ETOFormsProgressSection.tsx]: Please unify variables casing. `EtoKeyIndividualsProgress ` all previous variables are starting with lowercased letter

File: app/components/eto/dashboard/ETOFormsProgressSection.tsx [hunk 3]
```diff
@@ -42,50 +46,59 @@ class ETOFormsProgressSectionWidget extends React.Component<IProps> {
...
-          </Link>
+        <Col lg={4} xs={12} className="pt-2">
+          <EtoFormProgressWidget
+            isLoading={loadingData}
+            to={etoRegisterRoutes.productVision}
+            progress={shouldEtoDataLoad ? productVisionProgress : 0}
+            name="Product Vision"
+          />
         </Col>
         {/* TODO: ADD PROPER LOADING INDICATOR ONCE CONNECTED TO COMPONENTS */}
```
Comment (krzkaczor) [inline app/components/eto/dashboard/ETOFormsProgressSection.tsx]: is this todo still valid?

File: app/components/eto/dashboard/ETOFormsProgressSection.tsx (hunk 2 above)
Comment (Moejoe90) [reply app/components/eto/dashboard/ETOFormsProgressSection.tsx]: you are correct

File: app/components/eto/dashboard/ETOFormsProgressSection.tsx (hunk 3 above)
Comment (Moejoe90) [reply app/components/eto/dashboard/ETOFormsProgressSection.tsx]: good point

**Pull request merged**